src/components/photo/PhotoUploader.js

```diff
@@ -0,0 +1,55 @@
+import { equals } from 'ramda'
+import { useEffect, useState } from 'react'
+
+import { addPhoto } from '../../utils/api'
+import { PhotoOrganizer } from './PhotoOrganizer'
+
+export const arePhotosUploaded = (photos) =>
```

Comment:
maybe we could call this `isEveryPhotoUploaded`, but i may just be overly opinionated on not using `are` as a prefix